cmd/metrics-prometheus-collector/cmd/cmd.go

```diff
@@ -212,6 +199,8 @@ func RunE(_ *cobra.Command, _ []string) error {
 	if err != nil {
 		return err
 	}
+	metrics.Col.IsLeaderElected.Store(!options.LeaderElection)
```

Comment:
Should probably add a comment, because it's not intuitive why we're doing this. Something like:

```golang
   // If leader election is enabled, that means we have multiple instances and we
   // want to set each instance to false initially, and allow leader election to 
   // elect our leader. Conversely, if leader election is not enabled, we should
   // set this to true, since we only expect a single instance which is the de facto
   // leader.
```